connect/runtime/src/main/java/org/apache/kafka/connect/util/KafkaBasedLog.java

```diff
@@ -76,6 +76,10 @@
     private static final Logger log = LoggerFactory.getLogger(KafkaBasedLog.class);
     private static final long CREATE_TOPIC_TIMEOUT_NS = TimeUnit.SECONDS.toNanos(30);
     private static final long MAX_SLEEP_MS = TimeUnit.SECONDS.toMillis(1);
+    // 15min of admin retry duration to ensure successful metadata propagation.  10 seconds of backoff
+    // in between retries
+    private static final Duration ADMIN_CLIENT_RETRY_DURATION = Duration.ofMinutes(15);
+    private static final long ADMIN_CLIENT_RETRY_BACKOFF_MS = 10 * 1000;
```

Comment:
Nit: our convention is to use either `TimeUnit` or `Duration` to compute the millis to make it more readable and to reduce risk of conversion errors. Using `Duration` is a bit more readable, but the lines above already use `TimeUnit`:
```suggestion
    private static final long ADMIN_CLIENT_RETRY_BACKOFF_MS = TimeUnit.SECONDS.toMillis(10);
```